blockchain/common/common.go

```diff
@@ -1,23 +1,13 @@
 // Package blockchain provides functionality to interact with
 // different blockchain interfaces.
-package blockchain
+package common
```

Comment:
Is this a reason for this to be separate from `blockchain.go`? Apart from some naming conflicts with the other `common` package, `common.FMRequestState` for e.g might be slightly more understandable the old way, `blockchain.FMRequestState`?

Reply:
The reason for separating this out into a separate package was to avoid cyclic imports.

What we can do is to dot-import this package, so blockchain implementations could use it as just e.g. `FMRequestState` instead of `[package-name].FMRequestState`. If that sounds good, I'll have that done in the next PR.

Reply:
Good idea, yes